src/schemathesis/runner/__init__.py

```diff
@@ -276,6 +312,7 @@ def load_schema(
             # If `schema` is not an existing filesystem path or an URL then it is considered as an endpoint with
             # the given app
             loader = loaders.get_loader_for_app(app)
+            loader_options.update(dict_true_values(headers=headers))
```

Comment:
Hm, this change seems unrelated to the ASGI itself. On the first glance, it seems like we forgot to pass headers to loaders. Is it so? If yes, then it deserves a separate PR

Reply:
Yes, it is not must for asgi.

Reply:
Would you create a separate PR for that?

Reply:
Of course, it's a great honor. But the code test  failed on my other computer, and I'm looking for the reason, it may take some time

Reply:
Sure, thank you very much! :)